syntax = "proto3";

message ProfileCreate {
  string username = 1;
  string email = 2;
  int32 age = 3;
}

message ProfileGet {
  string id = 1;
  string username = 2;
  string email = 3;
  int32 age = 4;
  string created_at = 5;
}
